Ten Elephants/Ten Elephants/Controllers/MealPageController.swift

```diff
@@ -76,6 +77,17 @@ final class MealPageController: UIViewController {
         self.setNeedsStatusBarAppearanceUpdate()
     }
     
+    //MARK: - setLiked()
+    //executes when like button is pressed
+    @objc private func setLiked(_ sender: LikeButton){
```

Comment:
Nit: "setLiked" name suggests actions of marking something as liked, maybe something with "change" or "update" would fit better, what's your opinion?

Reply:
Yep, "update" fits better here. I'll rename it. Thanks!